src/visualizers/panels/OperationDepEditor/OperationDepEditorPanel.js

```diff
@@ -38,16 +38,18 @@ define([
         this.logger.debug('ctor finished');
     }
 
-    OperationDepEditorPanel.prototype = Object.create(TextEditorPanel.prototype);
+    OperationDepEditorPanel.prototype = Object.create(MonacoEditorPanel.prototype);
 
     OperationDepEditorPanel.prototype._initialize = function () {
         var self = this;
 
         //set Widget title
         this.setTitle('');
 
-        const config = {language: 'yaml'};
-        this.widget = new TextEditorWidget(this.logger, this.$el, config);
+        const config = {language: 'yaml',
+                        modelURI: 'inmemory://env.yml',
```

Comment:
These types of settings seem like they should probably be the defaults. In the monaco editor, I would just merge the provided options with the defaults. Something like:
```javascript
config = _.extend({}, DEFAULT_CONFIG, config);
```